src/database/PlayerInstance.ts

```diff
@@ -25,6 +27,18 @@ export class PlayerInstance {
     this.discordSnowflake = data.discordSnowflake;
     this.minecraftAccounts = data.minecraftAccounts;
     this.primaryMinecraftAccount = data.primaryMinecraftAccount ?? undefined;
+
+    PlayerInstance.instances.push(this);
+  }
+
+  public static resetAll() {
+    PlayerInstance.instances = [];
+  }
+
+  public static removePlayerInstance(instance: PlayerInstance) {
+    PlayerInstance.instances = PlayerInstance.instances.filter(
+      (i) => i !== instance
+    );
```

Comment:
this isn't needed, GameInstance already keeps track of all of the PlayerInstances.

Reply:
Done